compiler/ir/ir.tree/gen/org/jetbrains/kotlin/ir/expressions/IrGetValue.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+//This file was generated automatically
+//DO NOT MODIFY IT MANUALLY
+
+package org.jetbrains.kotlin.ir.expressions
+
+import org.jetbrains.kotlin.ir.visitors.IrElementVisitor
+
+/**
+ * A leaf IR tree element.
+ * @sample org.jetbrains.kotlin.ir.generator.IrTree.getValue
+ */
+abstract class IrGetValue : IrValueAccessExpression() {
+    override fun <R, D> accept(visitor: IrElementVisitor<R, D>, data: D): R =
+        visitor.visitGetValue(this, data)
+
+    abstract fun copyWithOffsets(newStartOffset: Int, newEndOffset: Int): IrGetValue
```

Comment:
Minor: probably it could be extracted to extension function, same as for IrConst